### YamlMime:ManagedReference
items:
- uid: System.IdentityModel.SignatureVerificationFailedException
  id: SignatureVerificationFailedException
  children:
  - System.IdentityModel.SignatureVerificationFailedException.#ctor
  - System.IdentityModel.SignatureVerificationFailedException.#ctor(System.String)
  - System.IdentityModel.SignatureVerificationFailedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.IdentityModel.SignatureVerificationFailedException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: SignatureVerificationFailedException
  nameWithType: SignatureVerificationFailedException
  fullName: System.IdentityModel.SignatureVerificationFailedException
  type: Class
  summary: "Wyjątek zgłaszany, gdy wystąpi błąd podczas przetwarzania podpisu"
  syntax:
    content: 'public class SignatureVerificationFailedException : System.IdentityModel.Tokens.SecurityTokenException'
  inheritance:
  - System.Exception
  - System.SystemException
  - System.IdentityModel.Tokens.SecurityTokenException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor
  id: '#ctor'
  parent: System.IdentityModel.SignatureVerificationFailedException
  langs:
  - csharp
  name: SignatureVerificationFailedException()
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException()
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException()
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Inicjuje nowe wystąpienie klasy <xref href=&quot;System.IdentityModel.SignatureVerificationFailedException&quot;> </xref> klasy."
  remarks: "Ten konstruktor inicjuje <xref:System.Exception.Message%2A>Właściwość nowego wystąpienia systemu dostarczanego przez komunikatu opisującego błąd.</xref:System.Exception.Message%2A> Ten komunikat bierze pod uwagę bieżące ustawienia kulturowe systemu.       W poniższej tabeli przedstawiono początkowe wartości właściwości dla nowego wystąpienia.      | Właściwość | Wartość |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| Ciąg z komunikatem o zlokalizowane. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public SignatureVerificationFailedException ();
    parameters: []
  overload: System.IdentityModel.SignatureVerificationFailedException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.IdentityModel.SignatureVerificationFailedException
  langs:
  - csharp
  name: SignatureVerificationFailedException(String)
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException(String)
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException(String)
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Inicjuje nowe wystąpienie klasy <xref href=&quot;System.IdentityModel.SignatureVerificationFailedException&quot;> </xref> klasy z powodu określonego błędu."
  remarks: "Ten konstruktor inicjuje <xref:System.Exception.Message%2A>Właściwość nowego przy użyciu wyjątek `message` parametru.</xref:System.Exception.Message%2A> Zawartość `message` ma na celu rozumiane przez człowieka. Element wywołujący tego konstruktora jest wymagana, aby upewnić się, że ten ciąg został zlokalizowany dla bieżącego ustawienia kulturowego systemu.       W poniższej tabeli przedstawiono początkowe wartości właściwości dla nowego wystąpienia.      | Właściwość | Wartość |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| Ciąg z komunikatem o. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public SignatureVerificationFailedException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Komunikat o błędzie opisujący przyczynę wyjątek."
  overload: System.IdentityModel.SignatureVerificationFailedException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.IdentityModel.SignatureVerificationFailedException
  langs:
  - csharp
  name: SignatureVerificationFailedException(SerializationInfo,StreamingContext)
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException(SerializationInfo,StreamingContext)
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Inicjuje nowe wystąpienie klasy <xref href=&quot;System.IdentityModel.SignatureVerificationFailedException&quot;> </xref> z zserializowanymi danymi."
  remarks: "Ten konstruktor jest wywoływana podczas deserializacji do odtworzenia obiekt wyjątku przesyłane przez strumień."
  syntax:
    content: protected SignatureVerificationFailedException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "A <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref> obiekt przechowujący dane serializacji obiektu."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "A <xref:System.Runtime.Serialization.StreamingContext>obiekt zawierający informacje kontekstowe dotyczące źródła lub miejsca docelowego.</xref:System.Runtime.Serialization.StreamingContext>"
  overload: System.IdentityModel.SignatureVerificationFailedException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.IdentityModel.SignatureVerificationFailedException
  langs:
  - csharp
  name: SignatureVerificationFailedException(String,Exception)
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException(String,Exception)
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException(String,Exception)
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Inicjuje nowe wystąpienie klasy <xref href=&quot;System.IdentityModel.SignatureVerificationFailedException&quot;> </xref> z określonego komunikatu o błędzie i odwołania do wyjątek wewnętrzny będący przyczyną tego wyjątku."
  remarks: "Wyjątek zgłaszany bezpośrednio w wyniku poprzedniego wyjątku powinien zawierać odwołanie do poprzedniego wyjątku we <xref:System.Exception.InnerException%2A>Właściwości.</xref:System.Exception.InnerException%2A> <xref:System.Exception.InnerException%2A>Właściwość zwraca tę samą wartość, która została przekazana do konstruktora.</xref:System.Exception.InnerException%2A>       W poniższej tabeli przedstawiono początkowe wartości właściwości dla nowego wystąpienia.      | Właściwość | Wartość |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Wyjątek wewnętrzny odwołania. |   | <xref:System.Exception.Message%2A>| Ciąg z komunikatem o. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public SignatureVerificationFailedException (string message, Exception inner);
    parameters:
    - id: message
      type: System.String
      description: "Komunikat o błędzie opisujący przyczynę wyjątek."
    - id: inner
      type: System.Exception
      description: "<xref:System.Exception>Będący przyczyną bieżącego wyjątku.</xref:System.Exception> Jeśli `inner` parametr nie jest <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, bieżący wyjątek jest zgłaszany w <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> bloku, który obsługuje wyjątek wewnętrzny."
  overload: System.IdentityModel.SignatureVerificationFailedException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.IdentityModel.Tokens.SecurityTokenException
  isExternal: false
  name: System.IdentityModel.Tokens.SecurityTokenException
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor
  parent: System.IdentityModel.SignatureVerificationFailedException
  isExternal: false
  name: SignatureVerificationFailedException()
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException()
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException()
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor(System.String)
  parent: System.IdentityModel.SignatureVerificationFailedException
  isExternal: false
  name: SignatureVerificationFailedException(String)
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException(String)
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.IdentityModel.SignatureVerificationFailedException
  isExternal: false
  name: SignatureVerificationFailedException(SerializationInfo,StreamingContext)
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException(SerializationInfo,StreamingContext)
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor(System.String,System.Exception)
  parent: System.IdentityModel.SignatureVerificationFailedException
  isExternal: false
  name: SignatureVerificationFailedException(String,Exception)
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException(String,Exception)
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor*
  parent: System.IdentityModel.SignatureVerificationFailedException
  isExternal: false
  name: SignatureVerificationFailedException
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException
